main.py

```diff
@@ -174,10 +174,10 @@ def main():
         global supported_currencies
 
         if announcement_coin not in order and announcement_coin not in sold_coins and announcement_coin not in old_coins:
-            logger.info(f'New annoucement detected: {announcement_coin}')
-
-            if supported_currencies is not False:
-                if announcement_coin in supported_currencies:
+            if announcement_coin:
+                logger.info(f'New annoucement detected: {announcement_coin}')
```

Comment:
`annoucement` is a misspelling. I was about to make a pull request myself, but I saw you already have one up, and what I do would just conflict.

I also think it'd be better to just check `announcement_coin` in the if statement on line 176, instead of adding it to a bunch of sub-if statements.

Reply:
I corrected the spelling error, but it's necessary to check all of those things on line 176. We need it to make sure we're not double-buying a coin.

Reply:
Thanks!

I meant to suggest to check it on line 176 in addition to all those other things, like the change you made for line 180.